pkg/initializer/initializer.go

```diff
@@ -35,6 +35,11 @@ import (
 	"k8s.io/client-go/util/retry"
 )
 
+const (
+	// retrySteps is the no. of steps used for exponential backoff to add a learner.
+	retrySteps = 6
```

Comment:
Out of curiosity, I am asking how we arrived at the number `6`?

Reply:
we are using `Backoff` with factor `2` with Duration set to `2 sec` , so I guess no. of retry `6` should be sufficient. WDYT?
https://github.com/gardener/etcd-backup-restore/blob/1ecfb1a55d9d1969dff6eb245e9e3a2de390d0bc/pkg/miscellaneous/miscellaneous.go#L584